### ISSUE TITLE
Loading Bar - Jumping back and forth when loading test level

### ISSUE THREAD

**Issue by dllllllllllb** [TheGameCreators/GameGuruRepo#2401]
Title

**Comment by dllllllllllb** [comment]
@LeeBamberTGC @PaulSJ  I moved the loading progress of "RASTERIZING NAVIGATION MESH" to Sample_TileMesh::buildAllTiles().

I kept the name of process the same as I'm not sure if it gets logged or is used for something. Also, if this is not a good place to update progress, let me know where I can move it to get it out of the way.

Due to Pauls wizardry, level loading is almost instant for me, even when loading our big demos, so I'm not 100% confident if it's working correctly.
